ts/features/itwallet/common/components/discoveryBanner/ItwDiscoveryBannerStandalone.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { ReactElement } from "react";
+import { useIOSelector } from "../../../../../store/hooks";
+import { isItwDiscoveryBannerRenderableSelector } from "../../store/selectors";
+import { ItwDiscoveryBanner } from "./ItwDiscoveryBanner";
+
+type ItwDiscoveryBannerProps = {
+  withTitle?: boolean;
+  ignoreMargins?: boolean;
+  fallbackComponent?: ReactElement;
+  closable?: boolean;
+};
```

Comment:
```suggestion
type FallbackProps = {
  fallbackComponent?: ReactElement;
};

type Props = Omit<ItwDiscoveryBannerProps, "handleOnClose"> & FallbackProps;
```

This could avoid repetitions. What do you think?